viewer/src/views/threejs/OrderSectorsByVisibilityCoverage.ts

```diff
@@ -124,7 +124,6 @@ export class GpuOrderSectorsByVisibilityCoverage {
     this.debugRenderer = new THREE.WebGLRenderer({
       antialias: false,
       alpha: false,
-      precision: 'lowp',
```

Comment:
Unrelated?

Reply:
Yes-ish. But just the debug view for the coverage helper...

Reply:
It's the `Also fixes a issue on iOS causing the new culler not do work optimally (accidentally committed and pushed to this branch and not its own...)`